react/features/base/participants/functions.js

```diff
@@ -1,3 +1,8 @@
+/* global MD5 */
+
+declare var config: Object;
+declare var interfaceConfig: Object;
```

Comment:
`global` and `declare var` are similar (with the added bonus that `declare var` will satisfy Flow as well so having flow type definitions for these external variable will one day allow us to be more strict about the types). So mixing them is against our goal of consistency. I'd prefer if we tried to be consistent at the file level at least. I'll change the `gloabl MD5` to `declare var MD5: Object`.